TeachingRecordSystem/src/TeachingRecordSystem.AuthorizeAccess/Pages/RequestTrn/DateOfBirth.cshtml

```diff
@@ -1,5 +1,23 @@
 @page "/request-trn/date-of-birth"
 @model TeachingRecordSystem.AuthorizeAccess.Pages.RequestTrn.DateOfBirthModel
 @{
-    ViewBag.Title = "What is your date of birth?";
+    ViewBag.Title = Html.DisplayNameFor(m => m.DateOfBirth);
 }
+
+@section BeforeContent {
+    <govuk-back-link href="@LinkGenerator.RequestTrnName(Model.JourneyInstance!.InstanceId)" />
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+        <form action="@LinkGenerator.RequestTrnDateOfBirth(Model.JourneyInstance!.InstanceId)" method="post">
+            <govuk-date-input asp-for="DateOfBirth">
```

Comment:
Would be good to set the autocomplete attributes on each part here - `bday-day`, `bday-month` and `bday-year`.
Something like `<govuk-date-input-day autocomplete="bday-day" />` I think will do it